wcs-importer-exporter.php

```diff
@@ -61,10 +64,31 @@ public static function setup_importer() {
 	}
 
 	/**
+	 * This is to calculate last_payment when the last payment not set by import
+	 * this is done by taking next interval and removing twice.
+	 * Only change from 0 if it was an imported subscription
+	 *
+	 * @param $date
+	 * @param $subscription @class WC_Subscription
+	 * @param $timezone
+	 * @return $date or date of last payment calulation
+	 */
+	public static function last_payment_calculation($date, $subscription, $timezone ) {
+		if ($date == 0 && "importer" === $subscription->__get("created_via")) {
```

Comment:
The `__get()` magic method shouldn't be called directly on `$subscription`, that defeats the purpose of the magic method. Instead, just use `$subscription->created_via` and the magic method will work its magic in the background. :)